packages/client-core/src/user/menus/ShareMenu.tsx

```diff
@@ -212,8 +212,8 @@ const ShareMenu = (): JSX.Element => {
           </div>
 
           <div className="flex w-fit items-center justify-center smh:w-full">
-            <div className="rounded-md bg-white p-4">
-              <QRCodeSVG className="h-[114px] w-[131px] smh:h-[161px] smh:w-[184px]" value={shareLink} />
```

Comment:
actually i had account for this dimensions by subtracting the padding (146-2*16). same for width

Reply:
Got it, Original dimensions according to designs are: 163x146. Subtracting 16px padding, we get: 147x130. 

147x130: 
![image](https://github.com/user-attachments/assets/6fa869c6-dead-4f38-af48-7c220ee3586c)

Designs:
![image](https://github.com/user-attachments/assets/218b6f7d-b8c8-4c3c-8322-eb2e3d1d01f3)